### ISSUE TITLE
ONIX display - other values - Additional accessibility information - Tactile

### ISSUE THREAD

**Issue by chrisONIX** [w3c/publ-a11y#320]
In the Display techniques for ONIX document  - Additional accessibility information  - Tactile section - what should be included in this section at the moment ?

**Comment by chrisONIX** [comment]
CROSSWALK _ https://w3c.github.io/a11y-discov-vocab/crosswalk/

schema.org  values tactileGraphic and tactileObject can not currently be mapped to anything in ONIX. 
This can be changed if there is use in the supply chain and therefore need of an ONIX code
